### ISSUE TITLE
Enhancements to RPM Packaging

### ISSUE THREAD

**Issue by spohnan** [ngageoint/geowave#342]
Some suggested improvements to the current RPM packaging:

* Create RPMs for cdh5 and apache vendor profiles
* Drop the vendor label geowave-**vendor**-rpmname for packages that don't differ by vendor
 * core
 * docs
 * source rpm
* Improve the "Installation from RPM" documentation to give examples of where RPMs should be applied
* Give instructions on how RPMs can be built locally using script we use for Jenkins (may require a bit of refactoring) 

__Example of Current RPM Packages__
```
geowave-cdh5-accumulo-0.8.6-201504141124.noarch.rpm
geowave-cdh5-core-0.8.6-201504141124.noarch.rpm
geowave-cdh5-docs-0.8.6-201504141124.noarch.rpm
geowave-cdh5-ingest-0.8.6-201504141124.noarch.rpm
geowave-cdh5-jetty-0.8.6-201504141124.noarch.rpm
geowave-cdh5-puppet-0.8.6-201504141124.noarch.rpm
geowave-cdh5-single-host-0.8.6-201504141124.noarch.rpm
geowave-cdh5-0.8.6-201504141124.src.rpm
```

__Would like to see__
```
geowave-puppet-0.8.6-201504141124.noarch.rpm
geowave-core-0.8.6-201504141124.noarch.rpm
geowave-docs-0.8.6-201504141124.noarch.rpm
geowave-apache-accumulo-0.8.6-201504141124.noarch.rpm
geowave-apache-ingest-0.8.6-201504141124.noarch.rpm
geowave-apache-jetty-0.8.6-201504141124.noarch.rpm
geowave-apache-single-host-0.8.6-201504141124.noarch.rpm
geowave-cdh5-accumulo-0.8.6-201504141124.noarch.rpm
geowave-cdh5-ingest-0.8.6-201504141124.noarch.rpm
geowave-cdh5-jetty-0.8.6-201504141124.noarch.rpm
geowave-cdh5-single-host-0.8.6-201504141124.noarch.rpm
geowave-0.8.6-201504141124.src.rpm
```

**Comment by rfecher** [comment]
With a recent merge, there will be implications on packaging.  In particular, the ingest framework plugs into an overarching command line tool, that other modules plug in to as well (analytics for example).  It seems to make sense to package the command line tool once rather than making it ingest specific.

Also, I went through the ascii docs fairly quickly and tried to note anything that looked like it would have a stale reference or need a different description on the packaging.  I started changing some but backed that out when I quickly realized it has implications on exactly how we intend to package the latest, particularly relating to the command line tool. Here's my list of files that seem to likely require a refresh:
* 041-statistics.adoc
* 050-framework.adoc
* 055-ingeststandalone.adoc
* 060-ingestplugins.adoc
* 075-install-from-rpm.adoc ("ingest" is probably not in the name of the GeoWaveMain commandline tool rpm anymore?)
* 080-install-from-source.adoc
* 085-accumulo-config.adoc ("ingest" is probably not in the name of the GeoWaveMain commandline tool rpm anymore?)
* 090-building.adoc (bottom section talking about avro)
* 095-jacejniproxies.adoc ("geowave-deploy" path references need to be replaced with deploy)

**Comment by spohnan** [comment]
This ticket is being closed and replaced with #370 and #371 to reflect what will be two different pull requests.

**Comment by jwomeara** [comment]
095-jacejniproxies updated with #379
